File: js/problem-1009.js
// https://www.beecrowd.com.br/judge/en/problems/view/1009

let input = require('fs').readFileSync('/dev/stdin', 'utf8');
let lines = input.split('\n');
let name = lines.shift();
let [fixedSalary, sale] =  lines.map(value => Number(value));
let salary = fixedSalary + (sale * 0.15);
console.log(`TOTAL = R$ ${salary.toFixed(2)}`);

File: cSharp/problem-1009.cs
using System; 

class URI {

    static void Main(string[] args) { 

            string name = Console.ReadLine();
            double fixedSalary = double.Parse(Console.ReadLine()),
            sale = double.Parse(Console.ReadLine()),
            total = fixedSalary + (sale * 0.15);

            Console.WriteLine($"TOTAL = R$ {total.ToString("n2").Replace(",", "")}");
    }

}